Makefile

```diff
@@ -12,19 +12,15 @@ GOPATH ?= $$HOME/go
 
 PROTOC_FLAGS := -I=. -I=./vendor -I=$(GOPATH)/src
 
-all: deps build test
+all: deps test
 
 dist:
-	cd cmd/bnsd ; make dist ; cd -
-	cd cmd/bcpd ; make dist ; cd -
+	cd cmd/bnsd && $(MAKE) dist
```

Comment:
👍

Reply:
DIfference between $(MAKE) and make?

Reply:
I guess it can be overwritten if you use non standard makefile :man_shrugging: